### DISCUSSION TITLE
Probe Y offset value

### DISCUSSION THREAD

**Discussion by JackTalisker** [Jyers/Marlin#1050] [Q&A] [answered]
I installed creality BLtouch and the E3V2-BLTouch-3x3-v4.2.2-v1.3.4 Jyers firmware which sets -7.0 as default value for the "Probe Y offset".
I realized that most of the online guides to install/configure BLTouch with Jyers fw recommend -6.0 for Y offset.
What is the correct value?

Thank you!

**Comment by Reggi3** [comment, marked as answer]
Each style of mount is different, so you should probably check with wherever you got your probe mount, if that's not possible, use a set of calipers to measure the distance between the nozzle and the probe in Y.

**Reply by JackTalisker** [reply]
I used the Creality official BLTouch upgrade with their specific mount. With the official one most sites recommend -44 for X offset and -6 for Y offset. The Jyers firmware sets them to -44 / -7

**Reply by Jyers** [reply]
It honestly varies even with the stock mount. The bend that defines the shape of the mount and position of the BL Touch varies largely for each case as it is obviously not bent very precisely. Perhaps a perfectly strait mount would yield a -44 -6, mine personally has -45.5 -8. The values of -44 -7 I believe have the best chance of being correct as they account for a mount that is not quite perfectly bent, which will be most.

If you really want any kind of accuracy in your probe offset you need to measure it yourself, there's no way around it. You don't even need any tools. Put some tape on your bed and mark where the nozzle is positioned. Now go to the move menu and record the position of the nozzle. Then move the printhead until the probe is directly over the point you marked. The difference between these two points will be your correct probe offset.

**Reply by JackTalisker** [reply]
Thank you Jyers!

**Comment by smooreace** [comment]
There are dozens of videos out there on editing and compiling firmware
specifically targeting the install of a BLTouch
